### ISSUE TITLE
delete compartments

### ISSUE THREAD

**Issue by michaeldeistler** [jaxleyverse/jaxley#551]
See [here](https://github.com/jaxleyverse/jaxley/discussions/550)

**Comment by jnsbck** [comment]
This is something that we can do via #355. Move module to graph, modify graph, re-import.
Alternatively we'd need to add a ’cell.branch(0).delete()’.

**Comment by michaeldeistler** [comment]
Yes I agree this would be an elegant way to solve this. I do not think we will desparately need the `.delete()` method if #355 is merged.

**Comment by michaeldeistler** [comment]
Closing this because #355 is merged now.
